composer.json

```diff
@@ -2,13 +2,15 @@
     "name": "microsoft/kiota-http-guzzle",
     "description": "Kiota HTTP Request Adapter implementation",
     "type": "library",
+	"minimum-stability": "beta",
```

Comment:
Instead of globally changing this, we can do it for the specific package and its beta dependencies:
```
"open-telemetry/sdk": "^1.0.0beta1@beta",
"open-telemetry/api": "@beta",
"open-telemetry/context": "@beta",
"open-telemetry/sem-conv": "@beta"
```